0.data_analysis/plate_5_characteristics/nbconverted/plate_5_sc_feature_selected_camerons_agg_well_corr.py

```diff
@@ -41,11 +43,11 @@
 
 # Path to correlation class
 sys.path.append(
-    f"{root_dir}/0.data_analysis/plate_5_characteristics/utils"
+    f"{root_dir}/0.data_analysis/utils"
 )
 
 # Class for calculating correlations
-from CorrelatePlate import CorrelatePlate
+from CorrelateData import CorrelateData
```

Comment:
Recommend moving all imports to the start of the notebook.

Reply:
I would, but I have to append the path from the root_dir variable, which is assigned after the imports.